kapitan/utils.py

```diff
@@ -120,7 +121,7 @@ def render_jinja2_file(name, context, jinja2_filters=defaults.DEFAULT_JINJA2_FIL
     return env.get_template(filename).render(context)
 
 
-def render_jinja2(path, context, jinja2_filters=defaults.DEFAULT_JINJA2_FILTERS_PATH):
+def render_jinja2(path, context, search_paths=None, jinja2_filters=defaults.DEFAULT_JINJA2_FILTERS_PATH):
```

Comment:
To keep things backwards-compatible, maybe `search_paths` as 4th param? That way we don't have to change our tests either

Reply:
I should have done that in the first place, fixed that now.